### ISSUE TITLE
replace xhtml2pdf with weasyprint

### ISSUE THREAD

**Issue by domdinicola** [unicef/etools-issues#1144]

**Comment by domdinicola** [comment]
@csabadenes  are you working already on this?
xhtml2pdf released a version which is also py3 compatible (thanks @dpoirier).
so we can use that and don't move to weasyprint

**Comment by csabadenes** [comment]
@domdinicola Yes i am working on it, but i was working on the separate PDF service - https://github.com/unicef/etools-pdf-service - i think this will run as a separate docker image, so it will have no interference with etools. Sorry, i did not realize that the library in etools should be upgraded.

So yeah.. i see no issue with using the new version of xhtml2pdf in etools(and since etools is upgrading to python 3, i can take a look how hard this would be to do for the new PDF service.)

**Comment by csabadenes** [comment]
@domdinicola, @dpoirier ,  i have also tried the newest xhtml2pdf release in etools, i noticed some UTF-8 issues, i am looking into a solution.

**Comment by csabadenes** [comment]
@domdinicola @dpoirier  The utf8 issues should be solved by https://github.com/unicef/etools/pull/1196

**Comment by csabadenes** [comment]
I think, for now,  the upgraded xhtml2pdf should be fine for the main branches. I have tested out Weasyprint, and there are a lot of CSS issues which will require some time to fix. If we want to change to WeasyPrint, i can continue to work on it on the separate PdfService branch(es).
